src/py/covidr/app.py

```diff
@@ -1,25 +1,104 @@
 import os
 from pathlib import Path
+from functools import wraps
+
+from dotenv import find_dotenv, load_dotenv
+from flask import Flask, send_from_directory, redirect, session, url_for
+from authlib.integrations.flask_client import OAuth
+from urllib.parse import urlencode
 
 from covidr.config import DevelopmentConfig
-from flask import Flask, send_from_directory
+
+
+ENV_FILE = find_dotenv()
+if ENV_FILE:
+    load_dotenv(ENV_FILE)
+
+AUTH0_CALLBACK_URL = os.environ.get("AUTH0_CALLBACK_URL")
+AUTH0_CLIENT_ID = os.environ.get("AUTH0_CLIENT_ID")
+AUTH0_CLIENT_SECRET = os.environ.get("AUTH0_CLIENT_SECRET")
+AUTH0_DOMAIN = os.environ.get("AUTH0_DOMAIN")
+AUTH0_BASE_URL = "https://" + AUTH0_DOMAIN
+SECRET_KEY = os.environ.get("SECRET_KEY")
```

Comment:
I suspect many of these could live in the Config object.

Reply:
+1